elastic/kibana
# [Stack Monitoring] Fix blank page between loading page and overview

## Summary

Fixes #112062

Changes `<RouteInit>` to render a `<LoadingPage>` while waiting for the cluster request, instead of rendering `null`.

### To test

1. Connect Kibana to a cloud Elasticsearch cluster. Running against a local cluster will load the requests too fast, and you won't notice the loading screens.
2. Start running Stack Monitoring. Just enabling self-monitoring is fine for this purpose, Metricbeat not needed.
3. Open either the Kibana homepage or any Kibana app that is not Stack Monitoring.
4. Navigate to Stack Monitoring from the sidebar.
5. Ensure that there is **always** a Loading spinner being displayed until the app has finished loading, instead of ever displaying a blank page.
6. Refresh the page, and ensure that there is still always a Loading spinner being displayed until the app has finished loading, instead of a ever displaying a blank page.
